static/app/views/replays/detail/console/index.tsx

```diff
@@ -94,9 +71,9 @@ function Console({breadcrumbs, startTimestampMs = 0}: Props) {
           {filteredBreadcrumbs.map((breadcrumb, i) => {
             return (
               <ConsoleMessage
-                isActive={i >= activeConsoleBounds[0] && i <= activeConsoleBounds[1]}
+                isActive={closestUserAction?.id === breadcrumb.id}
```

Comment:
> Fixing this `Crumb` thing seems like a whole separate issue. It looks like we should be [starting with ReplayReader](https://github.com/thisdot/sentry/blob/45fc69c8f7e4d4bb857f6677de769ff5a985a144/static/app/utils/replays/replayReader.tsx#L20) and working outward to fix a lot of places.
> 
> Maybe we should reduce the amount of change here to prevent extra merge conflicts with that bigger clearup

@ryan953 so I made this change because I needed the id of the breadcrumb to check when it was active - Should I look for a workaround?

Reply:
naw, it's all good